cocos/platform/mac/CCGLViewImpl-mac.mm

```diff
@@ -446,19 +446,18 @@ of this software and associated documentation files (the "Software"), to deal
 
 void GLViewImpl::enableRetina(bool enabled)
 {
-    //TODO coulsonwang
-//#if (CC_TARGET_PLATFORM == CC_PLATFORM_MAC)
-//    _isRetinaEnabled = enabled;
-//    if (_isRetinaEnabled)
-//    {
-//        _retinaFactor = 1;
-//    }
-//    else
-//    {
-//        _retinaFactor = 2;
-//    }
-//    updateFrameSize();
-//#endif
+#if (CC_TARGET_PLATFORM == CC_PLATFORM_MAC)
```

Comment:
Don't need the macro.

Reply:
yes, it's a mac specific implementation. I will remove the macro later.